### ISSUE TITLE
Size Mismatch Error using GPT large

### ISSUE THREAD

**Issue by shashank140195** [microsoft/BioGPT#85]
Hi, I am trying to use BioGPT large to finetune my dataset and I have replaced the --arch transformer_lm_prompt_biogpt_large in the train.sh file.  But I am still getting the mismatch error. I believe this is due to the wring size of vocab but I don't know where to change it as I was not able to find the right file for it. 

![image](https://user-images.githubusercontent.com/69673535/227040985-2e553195-f155-45eb-9333-2d07dcc4ece9.png)

**Comment by rezabuughunter** [comment]
The problem is with the file, check it

**Comment by rezabuughunter** [comment]
The problem is with the file, check it

**Comment by shashank140195** [comment]
> The problem is with the file, check it

Yeah, I checked my preprocessing bash file and I did not include large bpecodes and dict.txt for preprocessing data. I made the changes. though that error is resolved I am getting somewhat of a similar error. Do I need to make more changes to use a large BioGPT? 

![Screenshot 2023-03-23 at 12 04 23 PM](https://user-images.githubusercontent.com/69673535/227263947-37c3aee4-3dfe-46d8-8a44-98fc865ae406.png)

Please find the attached preprocess and train bash file and can you please tell me if I am doing something wrong or need to modify anything else?? 
[Archive 3.zip](https://github.com/microsoft/BioGPT/files/11053132/Archive.3.zip)

**Comment by shashank140195** [comment]
It says cannot load the model parameters from the checkpoint. Isn't my model downloaded successfully?
